### ISSUE TITLE
BUG: mdx-deck doesn't seem to work

### ISSUE THREAD

**Issue by EricSimons** [stackblitz/webcontainer-core#92]
---
name: Bug report
about: Create a report to help us improve
title: 'mdx-deck doesn't seem to work '
labels: ''
assignees: ''

---

**Describe the bug**

MDX deck seems to break- try running `npm run start` in the project. Seems to use gatsby under the hood so maybe related to #4 ?

This is the specific error it's throwing in the terminal:

![Screen Shot 2021-05-25 at 9 12 35 AM](https://user-images.githubusercontent.com/556934/119532027-5ca47780-bd39-11eb-9793-50433b3bb382.png)

**Link to the blitz that caused the error**
https://stackblitz.com/edit/mdx-deck?file=package.json

**To Reproduce**

**Expected behavior**

**Version of webcontainer**
Hash: 7286dbe8d47d3473cde7be488b7d048fb59b05f2

**Screenshots**

**Desktop (please complete the following information):**
Browser name  = Chrome 
Full version  = 90.0.4430.212
Major version = 90
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36
performance.memory = {
  totalJSHeapSize: 71231247,
  usedJSHeapSize: 56440659,
  jsHeapSizeLimit: 4294705152
}

**Additional context**

**Comment by junghyeonsu** [comment]
Is this issue still not resolved? or resolved?

**Comment by jrvidal** [comment]
@junghyeonsu Are you observing the same problem? If so, what's the project link? If a different one, would you mind opening a new issue?

**Comment by junghyeonsu** [comment]
Thank you! @jrvidal 
https://stackblitz.com/edit/node-sqwwbl?file=README.md,package.json
Gatsby version is v4.23.1

Dependency installation succeeds, but the start command fails.

**Comment by jrvidal** [comment]
Yep, I can reproduce. I've added the issue to our internal tracker. Thanks for reporting!
